run-explainer.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+sbt "-Dconfig.file=$HOME/.gu/explainer.local.conf" "run 9035"
```

Comment:
I've found this to be really good for config, don't know if you've seen it? : https://github.com/guardian/configuration-magic.